### PR TITLE
Add Brazilian Portuguese locale

### PR DESCRIPTION
**Pull request by jgalat**
This adds the Brazilian Portuguese locale file. 
I used the current Spanish locale file as template.

### CODE REVIEW

File: src/locale/pt-br.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import dayjs from 'dayjs'
+
+const locale = {
+  name: 'pt-br',
+  weekdays: 'Domingo_Segunda-feira_Terça-feira_Quarta-feira_Quinta-feira_Sexta-feira_Sábado'.split('_'),
+  months: 'Janeiro_Fevereiro_Março_Abril_Maio_Junho_Julho_Agosto_Setembro_Outubro_Novembro_Dezembro'.split('_'),
+  ordinal: n => n
```
Comment (iamkun) [inline src/locale/pt-br.js]: should this be `ordinal : '%dº'` ?  Not sure.

**Comment by iamkun** [comment]
Looks good thx

File: src/locale/pt-br.js (hunk 1 above)
Comment (jgalat) [reply src/locale/pt-br.js]: @iamkun The ordinal can be used or not. Here you can check examples that show that both ways are used in Brazil:
https://en.wikipedia.org/wiki/Date_and_time_notation_in_Brazil
http://www.maujor.com/w3c/datetime.html Here search for _Exemplos_ (Examples)

Comment (iamkun) [reply src/locale/pt-br.js]: After checking locale files in other JS date libraries, seems most are `nº`. But still it's up to you, native speaker. 😁

Comment (jgalat) [reply src/locale/pt-br.js]: @iamkun I've amended the commit with the fix. 
Thanks for the review!

Comment (iamkun) [reply src/locale/pt-br.js]: great THX

**Pull request merged**